dashboard/test/ui/features/blockTrashing.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Blocks can be trashed in certain circumstances
+
+Background:
+  Given I am on "http://learn.code.org/s/course2/stage/19/puzzle/2"
+  # In this stage:
+  # block 18 is the "when run" block
+  # block 19 is an undeletable "set color" block
+  # block 24 is a deletable "turn right" block
```

Comment:
in the scenario below:
````
When I drag block "turn right" to offset "-500, 0"
````

in the step definitions in /support/blockly_helpers.rb add something like:
````
def block_name_to_id(name)
  case nice
    when "when run"
      18
    when "set color"
      19
    when "turn right"
      24
    else
      name # we don't know what this is, just pass it through
    end
end
````
and in that above file and in step_definitions/blockly.rb refactor all usages of block ids to use the new method instead:

````
When /^I drag block "([^"]*)" to offset "([^"]*), ([^"]*)"$/ do |blockId, dx, dy|
  blockId = block_name_to_id(blockId)
  dragBlockRelative(blockId, dx, dy)
end
````